lib/utility/recipe_puppy_parser.rb

```diff
@@ -0,0 +1,92 @@
+module Utility
+  class RecipePuppyParser
+    ALL_RECIPES_URL = 'http://www.recipepuppy.com/api/?q=&p='.freeze
+    BASE_URL = 'http://www.recipepuppy.com/api/'.freeze
+
+# API/db scrape/validation PARENT 1
```

Comment:
Is this a permanent comment? It doesn't seem particularly helpful at first glance